127. Word Ladder/WordLadderStep3.java

```diff
@@ -0,0 +1,71 @@
+public class WordLadderStep3 {
+    // 12min
+    class Solution {
+        record WordAndCount(String word, int count) {}
+
+        public int ladderLength(String beginWord, String endWord, List<String> wordList) {
+            List<String> searchWords = new ArrayList<>();
+            searchWords.addAll(wordList);

```

Comment:
コンストラクタに渡しても良いかもしれません
```suggestion
            List<String> searchWords = new ArrayList(wordList);
```

Reply:
ありがとうございます。
こちらのほうがよいですね。